### PR TITLE
transformation: allow to do math

### PR DESCRIPTION
**Pull request by sileht**
This change introduces a math() transformer to do simple
mathematics on each points of the timeserie.

It uses numexpr library of pydata project.

Closes-bug: #342

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 6a212f8819ef49a4b7ed764b5f6d91aae160aa09

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 6f5974074f4a51bf27e2a13550c78169eab716f4

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 1b03ff737076879e8f0369dc9f3da04366ed94d3

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 11d845b1760128f75ddc2212fe4e0cff94b80489

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 6b1f1fcc482db7066ae44c26d2939b2f35eb6e3a

File: gnocchi/rest/transformation.py [hunk 1]
```diff
@@ -41,17 +42,42 @@ def transform(name, *args):
         lambda t: carbonara.Transformation(t[0], tuple(t[1])))
 
 
+def math_expression_parser(t):
+    # NOTE(sileht): ensure expression is valid
+    # We pass v into both dict to ensure evaluator don't use locals() and
+    # globals()
+    example = [1, 2, 3]
```
Comment (chungg) [inline gnocchi/rest/transformation.py]: uh... what's [1, 2, 3]?

**Comment by jd** [comment]
Closing in favor of https://github.com/gnocchixyz/gnocchi/pull/419

**Pull request closed without merging**